tests/VectorException/enc/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+oeedl_file(../VectorException.edl enclave gen)
+
 # TODO: Does this need CXX?
-add_enclave(TARGET VectorException_enc SOURCES enc.c sigill_handling.c init.cpp)
+add_enclave(TARGET VectorException_enc SOURCES
+    enc.c sigill_handling.c init.cpp ${gen})
```

Comment:
Reason for using CPP is because C doesn't support exception handling on Linux.

Reply:
I responded to the comment on line 6. TODO: Does this need CXX? It was in the original code.

Reply:
I responded to the comment on line 6. TODO: Does this need CXX? Didn't realize that this was in the original code.